.github/workflows/provenance.yaml

```diff
@@ -190,6 +198,40 @@ jobs:
       - name: Print SHA256 hash
         run: sha256sum oak_functions_freestanding_bin.intoto.jsonl
 
+    # Also post a reply on the PR thread with the sha256sum of the DSSE artifact.
+    - name: Post SHA256 Hash of the DSSE Artifact (post-merge only)
+      uses: actions/github-script@0.9.0
+      # if: github.event_name == 'push'
+      with:
+        github-token: ${{inputs.GITHUB_TOKEN}}
+        script: |
+          const execa = require('execa')
+          const { stdout } = await execa('sha256sum', ['oak_functions_freestanding_bin.intoto.jsonl'])
```

Comment:
How about `tee`ing the output of `ent` to a file, and then just read that file in the next step? Similar to what we do in the repro index case:

https://github.com/project-oak/oak/blob/e75740a22c5c3db3d25031cd8b660b50aa60eda2/.github/actions/reproducibility_index/action.yml#L43